pymagicc/io.py

```diff
@@ -2679,7 +2687,7 @@ def _write_variable_datablock_radforcing(self, output, data_block, units_level):
             data_block, units_level
         )
 
-        units = [u.replace("Wpermsuper2", "W/m2") for u in units]
+        units = [u.replace("W_per_msuper2", "W/m2") for u in units]
```

Comment:
Why `super` not `_super_` for consistency?

Reply:
I'm using "_" for space at the moment and I don't think pint would work if we had "W m ^ 2"

Reply:
although maybe it would...

Reply:
Ahh makes sense. It thought it was to tokenise it